python/fledge/services/core/server.py

```diff
@@ -604,6 +616,31 @@ async def core_logger_setup(cls):
             _logger.exception(ex)
             raise
 
+    @classmethod
+    async def setup_config_manager(cls):
+        """ Configuration manager category """
+        try:
+            if cls._configuration_manager is None:
+                cls._configuration_manager = ConfigurationManager(cls._storage_client_async)
+
+            config = cls._CONFIGURATION_DEFAULT_CONFIG
+            category = 'CONFIGURATION'
+            description = "Core Configuration Manager"
+            await cls._configuration_manager.create_category(category, config, description, True,
+                                                             display_name='Configuration Manager')
+            config = await cls._configuration_manager.get_category_all_items(category)
+            cache_size = int(config['cacheSize']['value'])
+            # Internal handling of cache size when with 0 and setting to default cache size of 30;
+            if cache_size == 0:
+                default_cache_size = cls._configuration_manager._cacheManager.max_cache_size
+                _logger.warn("Configuration Manager Cache Size is setting to default size of {}".format(
```

Comment:
Configuration Manager Cache Size is being set to the default size {}.
Setting Configuration Manager Cache Size to to the default size {}